src/SqlConverters.cs

```diff
@@ -192,7 +192,7 @@ public virtual async Task<string> BuildItemFromAttributeAsync(SqlAttribute attri
                     {
                         DateFormatString = ISO_8061_DATETIME_FORMAT
                     };
-                    return JsonConvert.SerializeObject(dataTable, jsonSerializerSettings);
+                    return Utils.SerializeObject(dataTable, jsonSerializerSettings);
```

Comment:
We have some places where we specify the settings already since we need to serialize dates in a specific way. I'll need to look at this more closely but we definitely don't want to be overwriting that here.